pom.xml

```diff
@@ -1208,6 +1328,7 @@
                             </goals>
                             <configuration>
                                 <classifier>${spark.version.classifier}tests</classifier>
+                                <skipIfEmpty>true</skipIfEmpty>
```

Comment:
Note, I changed this to note generate the tests jar is empty, this will affect other modules as well if you want me to split it out let me know.